.github/workflows/pr.yml

```
@@ -17,7 +17,7 @@ jobs:
     - name: golangci-lint
       uses: golangci/golangci-lint-action@v3
       with:
-        version: latest
+        version: '1.51.2'
```

Looks like the action requires `v` in the version string:
```suggestion
        version: v1.51.2
```